src/drive/web/modules/queries.js

```diff
@@ -355,3 +355,17 @@ export const buildFolderByPathQuery = path => ({
     fetchPolicy: defaultFetchPolicy
   }
 })
+
+export const buildNewSharingShortcutQuery = () => ({
+  definition: () =>
+    Q('io.cozy.files')
+      .where({
+        class: 'shortcut',
+        'metadata.sharing.status': 'new'
+      })
+      .indexFields(['class', 'metadata.sharing.status']),
+  options: {
+    as: 'io.cozy.files/new_sharing_shortcut',
```

Comment:
`as` should be "technically named" not "functionally named". Because the name is used to reuse a request, it's easier to reuse the same request if the name is technical instead of functional.

See https://github.com/cozy/cozy-guidelines/#naming-of-queries 

Here something like: `io.cozy.files/metadata.sharing.status/new/class/shortcut`